### ISSUE TITLE
*OTHER* Asthma Annual Review

### ISSUE THREAD

**Issue by richiecroker** [opensafely/codelist-development#73]
https://codelists.opensafely.org/codelist/user/richard-croker/asthma-annual-review/5a1d4594/

Currently only "annual review" codes included - other "review/monitoring" codes excluded.  To be agreed.

**Comment by brianmackenna** [comment]
How does this line-up with [QOF asthma review codes](https://cks.nice.org.uk/topics/asthma/goals-outcome-measures/qof-indicators/)?

We should probably use QOF list but very valuable to do full assessment of all codes available and see if any being used outside of QOF.

**Comment by richiecroker** [comment]
New codelist generated using [QOF Business Rules](https://digital.nhs.uk/data-and-information/data-collections-and-data-sets/data-collections/quality-and-outcomes-framework-qof/quality-and-outcome-framework-qof-business-rules/quality-and-outcomes-framework-qof-business-rules-v44-2019-2020-october-2020-release) reference set, codes obtained from [here](https://digital.nhs.uk/data-and-information/data-collections-and-data-sets/data-collections/quality-and-outcomes-framework-qof/quality-and-outcome-framework-qof-business-rules/primary-care-domain-reference-set-portal).

https://codelists.opensafely.org/codelist/user/richard-croker/asthma-annual-review-qof/33eeb7da/

**Comment by brianmackenna** [comment]
This looks good.

Question - for sake of QOF does appearance of any of these codes count for an "_asthma review_" ?  

The [QOF indicator on NICE states](https://cks.nice.org.uk/topics/asthma/goals-outcome-measures/qof-indicators/) _AST003 The percentage of patients with asthma, on the register, who have had an asthma review in the preceding 12 months that includes an assessment of asthma control using the 3 RCP questions_

**Comment by richiecroker** [comment]
Looking at the business rules, it appears that it's any of the above codes within the correct dates (minus exclusions due to unsuitability)
![image](https://user-images.githubusercontent.com/18683239/108870354-f439e300-75ef-11eb-9a8a-27ed3c334aff.png)
![image](https://user-images.githubusercontent.com/18683239/108870441-0582ef80-75f0-11eb-86cc-1346c7ae878e.png)
![image](https://user-images.githubusercontent.com/18683239/108870476-0fa4ee00-75f0-11eb-939f-6a2a0950a94f.png)

**Comment by richiecroker** [comment]
So, @brianmackenna do we need to add these codes as well?
![image](https://user-images.githubusercontent.com/18683239/108871037-a7a2d780-75f0-11eb-80e7-d4e3e8719c65.png)

**Comment by richiecroker** [comment]
agreed with @brianmackenna - no need to implement above three codes as only applicable for QoF validation at this time,

**Comment by richiecroker** [comment]
https://codelists.opensafely.org/codelist/user/richard-croker/asthma-annual-review-qof/33eeb7da/

**Comment by richiecroker** [comment]
@brianmackenna @inglesp i've been looking at this again.  The QoF indicator changed in 2020, and replaced the RCGP 3 questions marker with:
>The percentage of patients with asthma on the register, who have had an asthma review in the preceding 12 months that includes an assessment of asthma control using a validated asthma control questionnaire, a recording of the number of 
 exacerbations, an assessment of inhaler technique and a written personalised action plan.

The QoF business rules did change as rule, requiring those three things to be recorded on the same day in order to achieve the marker.

The issue is that, although the QoF marker changed for 2020-2021, asthma markers were suspended from QoF in 2020-2021 altogether, and didn't restart until April 2021.

I asked a GP colleague whether they'd switched to the new QoF measurements for 2020, and his response was:

>Tricky question to answer. QOF certainly moved last april 2020 to include plans etc and so we tried last year but I suspect most of us failed due to working remotely.

I have asked @HelenCEBM to run a quick count of the new codes (from refset `^999019931000230100`) but should we include new codes AND old codes for monitoring asthma reviews in 2020?
